scripts/release/download-gh-artifacts.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+RELEASE=$1
+SIGNER=$2
+
+echo "RELEASE: $RELEASE"
+echo "SIGNER: $SIGNER"
+
+for com in gh gpg curl sha256sum ; do
+	command -V ${com} >/dev/null 2>&1
+done
+
+[ ! -e "gh-release-artifacts/${RELEASE}" ]
+
+mkdir -p "gh-release-artifacts/${RELEASE}"
+cd "gh-release-artifacts/${RELEASE}"
+
+# github
+gh release download $RELEASE
```

Comment:
does this mean that a release has to be published on github before we can download it?

Reply:
No, the `gh` release download works with release drafts!